README.md

```diff
@@ -82,7 +85,7 @@ $> cd quorum-test-network
 Alternatively, you can use cli options and skip the prompt above like so:
 
 ```
-npx quorum-dev-quickstart -- --clientType besu --outputPath ./quorum-test-network --monitoring default --privacy true --orchestrate false
+npx quorum-dev-quickstart --clientType besu --outputPath ./quorum-test-network --monitoring default --privacy true --orchestrate false
```

Comment:
We need to keep this because yargs requires the extra two `--` in front of the options we pass in

Reply:
when I tried it locally though it didn't work unless I removed the extra dashes
```
➜  workspace npx quorum-dev-quickstart -- --clientType besu --outputPath ./quorum-test-network --monitoring default --privacy true --orchestrate false
Options:
  --help         Show help                                             [boolean]
  --version      Show version number                                   [boolean]
  --clientType   Ethereum client to use.
                               [string] [required] [choices: "besu", "goquorum"]
  --outputPath   Location for config files.
                                     [string] [default: "./quorum-test-network"]
  --monitoring   Enable support for monitoring with Splunk or ELK.
                                                   [string] [default: "default"]
  --privacy      Enable support for private transactions    [boolean] [required]
  --orchestrate  Try out Codefi Orchestrate?           [boolean] [default: true]

Missing required arguments: clientType, privacy
➜  workspace npx quorum-dev-quickstart --clientType besu --outputPath ./quorum-test-network --monitoring default --privacy true --orchestrate false
✅ Installation complete.
```

Reply:
is this an os thing? I'm on Mac

Reply:
Actually seems to be a thing when you run it direct from source via npm run start

```
~/workspace/quorum-dev-quickstart(master)$ npm run start --clientType gquorum --privacy true 

> quorum-dev-quickstart@0.0.28 start /home/jfernandes/workspace/quorum-dev-quickstart
> node build/index.js "gquorum"

Options:
  --help         Show help                                             [boolean]
  --version      Show version number                                   [boolean]
  --clientType   Ethereum client to use.
                               [string] [required] [choices: "besu", "goquorum"]
  --outputPath   Location for config files.
                                     [string] [default: "./quorum-test-network"]
  --monitoring   Enable support for monitoring with Splunk or ELK.
                                                   [string] [default: "default"]
  --privacy      Enable support for private transactions
                                           [boolean] [required] [default: false]
  --orchestrate  Try out Codefi Orchestrate?          [boolean] [default: false]

Missing required argument: clientType

```

Given this is via npx in the docs, we can omit it. I'll approve and merge